lib/page/get-display-value/get-display-value.unit.spec.js

```diff
@@ -51,38 +51,38 @@ test('When there is a value', t => {
   {
     resetStubs()
 
-    const pageInstance = {skipRedact: true}
+    getDisplayValueStub.returns('display value')
 
-    const displayValue = getDisplayValue(pageInstance, userData, nameInstance)
+    const displayValue = getDisplayValue(userData, nameInstance)
 
     t.equals(displayValue, 'display value', 'it returns the display value')
-    t.ok(getRedactedValueStub.calledOnce, 'it calls `getRedactedValue` once')
-    t.deepEqual(getRedactedValueStub.getCall(0).args, [nameInstance, userData, true, 'input'], 'it calls `getRedactedValue` with the expected args (`skipRedact` is true)')
+    // t.ok(getRedactedValueStub.calledOnce, 'it calls `getRedactedValue` once')
```

Comment:
what went wrong here?

Reply:
Oh, good spot. That whole unit has changed -- this spec is (pretty much) no longer needed. All of the tests are moved into specs for the controllers, so that file (bar one or two lines) is commented out. I'll update it

Reply:
Done